ledger/simulation/trace.go

```diff
@@ -135,6 +141,13 @@ func validateSimulateRequest(request Request, developerAPI bool) error {
 			},
 		}
 	}
+	if !request.TraceConfig.Enable && request.TraceConfig.Stack {
+		return InvalidRequestError{
+			SimulatorError{
+				err: fmt.Errorf("the request didn't ask for enabling returning basic trace, but ask for returning stack change"),
```

Comment:
```suggestion
				err: fmt.Errorf("basic trace must be enabled when enabling stack tracing"),
```

Reply:
thanks for the change, see [per jjs comments](https://github.com/algorand/go-algorand/pull/5440/commits/b2bdb2d78d93d3f9338be0e418eeab99c19b4ef0)